### PR TITLE
Makes polar bears not become easier to move after you kill and revive them 

### LINKED ISSUE #56956: polar bears are easier to move after you kill and revive them
```
    move_force = MOVE_FORCE_VERY_STRONG
    move_resist = MOVE_FORCE_VERY_STRONG
    pull_force = MOVE_FORCE_VERY_STRONG
```

```
/mob/living/simple_animal/hostile/asteroid/polarbear/death(gibbed)
	move_force = MOVE_FORCE_DEFAULT
	move_resist = MOVE_RESIST_DEFAULT
	pull_force = PULL_FORCE_DEFAULT
	return ..()
```

https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/simple_animal/hostile/mining_mobs/polarbear.dm

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Fixes #56956.

## Why It's Good For The Game
bugfix.

## Changelog
:cl:
fix: Killing and reviving polar bears no longer makes them easier to move.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/polarbear.dm [hunk 1]
```diff
@@ -59,6 +59,12 @@
 	pull_force = PULL_FORCE_DEFAULT
 	return ..()
 
+/mob/living/simple_animal/hostile/asteroid/polarbear/revive(full_heal = FALSE, admin_revive = FALSE)
+	. = ..()
+	move_force = initial(move_force)
+	move_resist = initial(move_resist)
+	pull_force = initial(pull_force)
```
Comment (ATH1909) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/polarbear.dm:66]: won't this code run regardless of whether or not the revive attempt was successful?

Comment (interestingusernam3) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/polarbear.dm]: It will. Thanks for pointing this out.

**Comment by Mothblocks** [comment]
image
Hm

**Comment by interestingusernam3** [comment]
@Mothblocks Yes.

**Pull request merged**